.CONTRIBUTING.md

```diff
@@ -64,15 +64,43 @@ description: Install the software to compile your decentralized network, configu
 icon: octicons-code-24
 template: index-page.html
 ---
-```
 
+<!-- Add content in markdown here --> 
+
+## Explore What This Section Covers
+
+:::INSERT_GENERATED_CARDS:::
+```
 Some important things to note:
```

Comment:
Watch spacing here. There should be a space between the code sample and the following sentence